components/brave_wallet/common/hash_utils.cc

```diff
@@ -62,17 +66,42 @@ eth_abi::Bytes32 Namehash(std::string_view name) {
 }
 
 SHA256HashArray DoubleSHA256Hash(base::span<const uint8_t> input) {
-  return crypto::SHA256Hash(crypto::SHA256Hash(input));
+  return crypto::hash::Sha256(crypto::hash::Sha256(input));
 }
 
 Ripemd160HashArray Hash160(base::span<const uint8_t> input) {
   Ripemd160HashArray result = {};
 
   CRIPEMD160()
-      .Write(crypto::SHA256Hash(input).data(), crypto::kSHA256Length)
+      .Write(crypto::hash::Sha256(input).data(), crypto::hash::kSha256Size)
       .Finalize(result.data());
 
   return result;
 }
 
+std::vector<uint8_t> Blake2bHash(
```

Comment:
Could this be reused in the ZCashSerializer too?

Reply:
will do!